app/src/main/java/org/p2p/wallet/settings/interactor/SettingsInteractor.kt

```diff
@@ -13,7 +14,7 @@ class SettingsInteractor(
 ) {
 
     fun setZeroBalanceHidden(isHidden: Boolean) {
-        sharedPreferences.edit { putBoolean(KEY_HIDDEN_ZERO_BALANCE, isHidden) }
+        sharedPreferences.edit(commit = true) { putBoolean(KEY_HIDDEN_ZERO_BALANCE, isHidden) }
```

Comment:
Don't do that, it will cause to skipped frames
We should use it in very rare cases